tests/test_integration.py

```diff
@@ -416,6 +417,40 @@ def test_interpreter_resolution_with_pex_python_path():
       assert str(pex_python_path.split(':')[0]).encode() in stdout
 
 
+@pytest.mark.skipif(IS_PYPY)
+def test_interpreter_constraints_honored_without_ppp_or_pp():
+  # Create a pex with interpreter constraints, but for not the default interpreter in the path.
+  with temporary_dir() as td:
+    py36_path = ensure_python_interpreter(PY36)
+    py35_path = ensure_python_interpreter(PY35)
```

Comment:
Trivial nit: put py35 before py36, as the informal convention for Pants and Pex seems to be going from older to newer. Also applies to line 431 and 432.